inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/InlongGroupServiceImpl.java

```diff
@@ -206,6 +219,24 @@ public InlongGroupInfo get(String groupId) {
         Page<InlongGroupEntity> entityPage = (Page<InlongGroupEntity>) groupMapper.selectByCondition(request);
         List<InlongGroupListResponse> groupList = CommonBeanUtils.copyListProperties(entityPage,
                 InlongGroupListResponse::new);
+        if (request.isListSources() && CollectionUtils.isNotEmpty(groupList)) {
```

Comment:
There may be a lot of streams and sources under these groups, and the result set of the query may be very large.

Reply:
default listsources is false
@ApiModelProperty(value = "If list streamSource for group", hidden = true)
 private boolean listSources = false;